samples/cpp/ngraph_function_creation_sample/main.cpp

```diff
@@ -104,8 +76,8 @@ ov::runtime::Tensor ReadWeights(const std::string& filepath) {
  * @brief Create ngraph function
  * @return Ptr to ngraph function
  */
-std::shared_ptr<ov::Model> createNgraphFunction() {
-    auto weights = ReadWeights(FLAGS_m);
+std::shared_ptr<ov::Model> createNgraphFunction(const std::string& path_to_weights) {
```

Comment:
create_ov_function

Reply:
you see outdated diff. the actual one is:
std::shared_ptr<ov::Model> createNgraphFunction(const std::string& path_to_weights) {

Reply:
But in new API we don't have nGraph function only ov::Model. Will be better to have the name `create_ov_model`